di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/service/TokenService.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.di.ipv.stub.cred.service;
+
+import com.nimbusds.oauth2.sdk.token.AccessToken;
+import com.nimbusds.oauth2.sdk.token.BearerAccessToken;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class TokenService {
+
+    private static final long DEFAULT_ACCESS_TOKEN_TTL_IN_SECS = 3600;
+    private final Map<String, String> accessTokens;
+
+    public TokenService() {
+        this.accessTokens = new ConcurrentHashMap<>();
+    }
+
+    public void persist(AccessToken accessToken, String resourcePayload) {
```

Comment:
Same as the AuthCode equivalent. I think this would be better if the accessToken was a sting here, making the `put` and `get` symmetrical.

Not a blocker. If it causes any issues we can change it.